src/NUnitFramework/framework/nunit.framework-portable.csproj

```diff
@@ -186,6 +186,9 @@
     <Compile Include="Attributes\ValuesAttribute.cs" />
     <Compile Include="Attributes\ValueSourceAttribute.cs" />
     <Compile Include="CollectionAssert.cs" />
+    <Compile Include="Compatibility\System.Collections.Concurrent\ConcurrentQueue.cs" />
+    <Compile Include="Compatibility\System.Collections.Concurrent\IProducerConsumerCollection.cs" />
+    <Compile Include="Compatibility\System.Collections\CollectionDebuggerView.cs" />
```

Comment:
Not sure why we do this in portable, which doesn't even support parallel execution.